src/DynamoCoreWpf/ViewModels/Watch3D/AttachedProperties.cs

```diff
@@ -16,41 +16,16 @@ public static class AttachedProperties
 
         // handles determining color of elementGeometry3Ds when any property is set false
         // as the state of other properties must be checked to determine the correct color / material.
-        private static void OnPropertySetFalse(DependencyObject obj)
+        private static void OnPoint_Line_PropertySetFalse(DependencyObject obj)
```

Comment:
Just a minor thing but I think it is better to avoid underscore character for function names unless they are test functions. Makes it easy to differentiate. 
https://stackoverflow.com/questions/9782084/naming-conventions-in-c-sharp-underscores
https://docs.microsoft.com/en-us/dynamicsax-2012/developer/naming-conventions-underscores.

Reply:
thanks @reddyashish I will change the name.